afutests/reset/ocxl_reset_tests.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/bash
+#
+# Copyright 2020 International Business Machines
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# ocxl_reset_tests.sh
+#
+# This test assumes that user is root and memcpy afu is programmed.
+
+function usage
+{
+	echo 'ocxl_reset_tests.sh [-d <device_path>] [-l <loops>]' >&2
+	exit 2
+}
+
+device=
+loops=1 # default
+
+while true
+do
+	case $1 in
+	('')	break ;;
+	(-d)	device=$2; shift 2 || break ;;
+	(-l)	loops=$2; shift 2 || break ;;
+	(*)	usage ;;
+	esac
+done
+(( $# == 0 )) || usage
+
+[[ $device == -* ]] && usage
+(( loops < 1 )) && usage
+
+if [[ $device ]]
+then
+	if ! ls "$device" >/dev/null 2>&1
+	then
+		echo ocxl_reset_tests.sh: "$device": no such device >&2
+		exit 2
+	fi
+	card=${device##*/}	
+fi
+
+if [[ -z $card ]]
+then
+	# find first IBM,AFP3 or IBM,MEMCPY3 opencapi card
+	card=$(
+		set -- $(ls /dev/ocxl/$card 2>/dev/null)
```

Comment:
The $card here seems spurious, as $card is empty

Reply:
thanks